### DISCUSSION TITLE
Geeting "FFmpeg transcoding error"

### DISCUSSION THREAD

**Discussion by Laserstrahl** [audiamus/AaxAudioConverter#168] [Q&A] [answered]
When trying to convert an AAX I'm getting "Abort. 0 Files converted. FFmpeg Transcoding Error". What am I doing wrong?

**Comment by audiamus** [comment]
A log file might help to figure it out. Log level 3 should be sufficient at this stage. See the manual how to create log files.

**Comment by Laserstrahl** [comment, marked as answer]
Thanks for the reply. The Logfile indicated a DRM problem. I reinstalled AaxAudioConverter and BookLib. And created the BookLib Profile again. Now it works. Maybe something went wrong with BookLib in the first place, dunno.

The ticket can be closed.
